frontend/packages/knative-plugin/src/components/overview/OverviewDetailsKnativeResourcesTab.tsx

```diff
@@ -45,6 +46,18 @@ const getSidebarResources = ({ obj, ksroutes, revisions, configurations }: Overv
 };
 const OverviewDetailsKnativeResourcesTab: React.FC<OverviewDetailsResourcesTabProps> = ({
   item,
-}) => <div className="overview__sidebar-pane-body"> {getSidebarResources(item)} </div>;
+}) => (
+  <div className="overview__sidebar-pane-body">
```

Comment:
I see the same code repeated twice. Make this a pure component and share it in both places.